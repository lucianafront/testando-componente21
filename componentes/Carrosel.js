

function Carrosel() {
  return (
  <>
  

	<div className="underline-box ready" style={{marginTop: "35px", textAlign: "center"}}>
		<h1 className="rising-reveal"><span className="letter">B</span><span className="letter">O</span><span className="letter">L</span><span className="letter">O</span><span className="letter">S</span></h1>
		<p>Deliciosamente recheados e com muito sabor, cada um de nossos bolos é criado <br/> para proporcionar mais amor e carinho para todas as ocasiões.</p>
	</div>


  </>
  );
}

export default Carrosel;
