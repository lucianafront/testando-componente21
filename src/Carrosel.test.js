import React from 'react';
import { render } from '@testing-library/react';
import Carrosel from './componentes/Carrosel';

test('deve renderizar corretamente o componente Carrosel', () => {
  const { asFragment } = render(<Carrosel />);
  expect(asFragment()).toMatchSnapshot();
});
