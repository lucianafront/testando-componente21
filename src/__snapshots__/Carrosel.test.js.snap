// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`deve renderizar corretamente o componente Carrosel 1`] = `
<DocumentFragment>
  <div
    class="underline-box ready"
    style="margin-top: 35px; text-align: center;"
  >
    <h1
      class="rising-reveal"
    >
      <span
        class="letter"
      >
        B
      </span>
      <span
        class="letter"
      >
        O
      </span>
      <span
        class="letter"
      >
        L
      </span>
      <span
        class="letter"
      >
        O
      </span>
      <span
        class="letter"
      >
        S
      </span>
    </h1>
    <p>
      Deliciosamente recheados e com muito sabor, cada um de nossos bolos é criado 
      <br />
       para proporcionar mais amor e carinho para todas as ocasiões.
    </p>
  </div>
</DocumentFragment>
`;
